### ISSUE TITLE
Update slides to match current state of curriculum

### ISSUE THREAD

**Issue by bridgethillyer** [ClojureBridge/curriculum#90]
This is going to be tricky. We need a plan for how to keep these up to date.

**Comment by kpfell** [comment]
This issue references updating the slides, but have we discussed how we want to keep the slides up to date, long-term? A section in [CONTRIBUTING.md](https://github.com/ClojureBridge/curriculum/blob/master/CONTRIBUTING.md) is necessary at this point, IMHO.

**Comment by rubygeek** [comment]
I updated the slides to reflect the removal of heroku/webapp in this repo but mistakenly built it on top if austin clojure .. if the slides portions look good, i can separate the commits:   https://github.com/AustinClojure/curriculum/compare/slides-notes_remove-webapp

**Comment by yokolet** [comment]
Hi all,
I made a progress today. See, http://localhost:5555/experimenting-curriculum-ghpages/intro.html, I need to add css/javascript to make long sentences pop up, though.

The page's source is https://github.com/yokolet/experimenting-curriculum-ghpages/blob/gh-pages/intro.md

This markdown has some styling notations, but basically normal markdown. So people can make a pull request to the markdown file. Once the pull request is merged, updated slides will be there.

I'll keep working on this some more.

**Comment by yokolet** [comment]
Wow, wrong URL, sorry.
http://yokolet.github.io/experimenting-curriculum-ghpages/intro.html

**Comment by yokolet** [comment]
Now, my experimenting comes here.

This markdown file,
https://github.com/yokolet/experimenting-curriculum-ghpages/blob/gh-pages/intro.md
is rendered,
http://yokolet.github.io/experimenting-curriculum-ghpages/intro.html

I added some styling markdown notations and html tags in the file to make it slide-ish. Long sentences are hidden and show up when the button is clicked. But, still, the markdown file is readable as markdown on github.

I think we should move all curriculum contents to gh-pages branch and keep markdown files up to date. Then, a sync problem between outlines and slides will go away. Just pushing changes or merging pull requests update the slide on github pages. Besides, editing markdown is much easier than html.

If you think this is the right way, I'll move my work on curriculum repository an change other curriculum markdown files. How do you think?

**Comment by yokolet** [comment]
Over the weekend, I moved all contents to gh-pages and add styling things to curriculum markdown files. Now we can see the slides at http://clojurebridge.github.io/curriculum/ , which are rendered from markdown files in outline directory.

Also I added titles to each slides if missing. It was not apparent but on slides, no title pages looked weird. If those titles are not appropriate, open an issue or make a pull request.

Plus, I added some samples and words where didn't make sense. I guess some were deleted but explanations remained unedited.

As an editing guide, I added https://github.com/ClojureBridge/curriculum/blob/gh-pages/EDITING-CURRICULUM.md . So, have a look before editing or merging PR.

Now, curriculum markdown files are the source of slides and a sync problem's gone. I'm going to close this issue.
